pyproject.toml

```
@@ -7,6 +7,7 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = "^3.8"
+pytest = "^6.2.5"
```

pytestはビルド時に必要ないので、`dev-dependencies`に追加した方が良さそうです。

```suggestion
[tool.poetry.dev-dependencies]
pytest = "^6.2.5"
```

---

気づいてませんでした... ありがとうございます!